contrib/devtools/lint-include-guards.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+HEADER_ID_PREFIX="BITCOIN_"
+HEADER_ID_SUFFIX="_H"
+
+REGEXP_EXCLUDE_FILES_WITH_PREFIX="src/(config/bitcoin-config.h|crypto/ctaes/|leveldb/|obj/build.h|qt/forms/|secp256k1/|tinyformat.h|univalue/)"
```

Comment:
Do you really need bitcoin-config.h and build.h in here? They're auto-generated so I would have thought git ls-files wouldn't have listed them.